scribus/scpainter.cpp

```diff
@@ -1955,8 +1962,18 @@ void ScPainter::drawText(QRectF area, QString text, bool filled, int align)
 	double ww = 0;
 	double hh = 0;
 	double r, g, b;
-	QString fontName = m_font.family() + " " + m_font.style();
-	cairo_select_font_face(m_cr, fontName.toLatin1(), CAIRO_FONT_SLANT_NORMAL, CAIRO_FONT_WEIGHT_NORMAL);
+
+	if (m_font.isNone())
```

Comment:
if you call setFont once, there will always be an m_font even if it is the one you want, so that code at client side will need to be kept anyway.